ale_linters/scala/scalastyle.vim

```diff
@@ -0,0 +1,46 @@
+" Author: Kevin Kays - https://github.com/okkays
+" Description: Support for the scalastyle checker.
+
+let g:ale_scala_scalastyle_options =
+\   get(g:, 'ale_scala_scalastyle_options', '-c scalastyle-config.xml')
```

Comment:
Is that a common configuration file name? You should try and search for it in parent directories, and make it automatic. Look at the JSHint linter or other linters for examples.